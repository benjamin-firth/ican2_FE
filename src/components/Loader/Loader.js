import React from 'react';
import './Loader.scss';
import PropTypes from 'prop-types';

const Loader = ({ message }) => {

  return (
    <section className='loader-container'>
      <h5>{message}</h5>
    </section>
  );
};

Loader.propTypes = {
  message: PropTypes.string
}

export default Loader;
