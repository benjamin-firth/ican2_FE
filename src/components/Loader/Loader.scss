@import '../../variables.scss';

.loader-container {
  height: auto;
  min-height: 100vw;
  background-color: $primarycolor;
  padding-top: 7vw;
  text-align: center;
  h5 {
    color: $secondarycolor;
    font-family: $secondaryfont;
    font-size: 35px;
    margin: 8vh 0 0 10vw;
  }
}

@media only screen and (max-width: 400px) {
  .loader-container {
    height: auto;
    min-height: 195vw;
    padding-top: 50vw;
    h5 {
      font-size: 35px;
      margin: 8vh 0 0 10vw;
    }
  }
}
