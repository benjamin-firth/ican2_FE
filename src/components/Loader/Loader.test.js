import React from 'react';
import { shallow } from 'enzyme';
import Loader from './About';

describe('Loader', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow( <Loader />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
