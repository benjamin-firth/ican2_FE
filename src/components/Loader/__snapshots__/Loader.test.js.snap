// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match the snapshot 1`] = `
<section
  className="loader-container"
>
  <h5>
    Loading...
  </h5>
</section>
`;
